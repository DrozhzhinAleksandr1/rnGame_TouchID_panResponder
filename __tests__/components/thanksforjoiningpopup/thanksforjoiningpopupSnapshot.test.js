import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { ThanksForJoiningPopUp } from '../../../app/components/thanksforjoiningpopup/thanksforjoiningpopup';
import { localization } from '../../../app/common/localization';

it('create snapshot ThanksForJoiningPopUp', () => {
    const renderer = new ShallowRenderer()
    let thanksForJoiningPopUp = renderer.render(<ThanksForJoiningPopUp CurrentThanksForJoiningPopUpVisibility={true} CurrentLanguage={localization.ru}/>);
    expect(thanksForJoiningPopUp).toMatchSnapshot();
});