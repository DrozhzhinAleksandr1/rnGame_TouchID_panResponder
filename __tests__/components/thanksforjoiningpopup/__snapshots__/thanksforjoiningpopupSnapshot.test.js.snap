// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot ThanksForJoiningPopUp 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(23, 37, 51, 0.8)",
      "elevation": 10,
      "height": "100%",
      "justifyContent": "center",
      "paddingLeft": 16,
      "paddingRight": 16,
      "position": "absolute",
      "width": "100%",
      "zIndex": 10,
    }
  }
>
  <TouchableWithoutFeedback
    accessibilityLabel="BublePopupID"
    accessible={false}
    onPress={[Function]}
    testID="BublePopupID"
  >
    <Component
      style={
        Object {
          "backgroundColor": "transparent",
          "height": "100%",
          "position": "absolute",
          "width": 750,
        }
      }
    />
  </TouchableWithoutFeedback>
  <Component
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#1D2E3E",
        "justifyContent": "center",
        "shadowColor": "rgba(0, 0, 0, 0.7)",
        "shadowOffset": Object {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 1,
        "shadowRadius": 8,
        "width": "100%",
        "zIndex": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Regular",
          "fontSize": 20,
          "lineHeight": 23,
          "marginTop": 24,
          "textAlign": "center",
        }
      }
    >
      Спасибо,

 что присоеденились к нам!
    </Text>
    <Text
      accessibilityLabel="TextRegistrationPopupID"
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Light",
          "fontSize": 16,
          "lineHeight": 19,
          "marginTop": 10,
          "textAlign": "center",
          "width": 272,
        }
      }
      testID="TextRegistrationPopupID"
    >
      Письмо для подтверждения Ваших данных и завершения процесса регистрации отправлено на указаный e-mail
    </Text>
    <Component
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "center",
          "paddingBottom": 20,
          "paddingLeft": 24,
          "paddingRight": 24,
          "width": "100%",
        }
      }
    >
      <GreenPopUpBtn
        func={[Function]}
        text="ОК"
      />
    </Component>
  </Component>
</Component>
`;
